### PR TITLE
Staging/dashboardv2

### PR DESCRIPTION
**Pull request by MohiTheFish**
Waiting on designs for some of the cells, but the time/clock, events, logo, and twitter feed are looking good.

### CODE REVIEW

**Review by brianstrauch — COMMENTED** [review]
Looks good! A few things we should change:
- Display more tweets (or auto-scroll them somehow)
- Countdown clock is off by a minute
- Events are getting cut off at the bottom of the page
- I'd remove the "Countdown" and "Time" <h1> tags because it's kind of obvious what they are

File: src/scenes/Dashboard/cells/countdown.jsx [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    hackillinoisStartTime,
+    hackillinoisEndTime,
+  };
+}
+
+// This component is in the top row, center
+class CountDown extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = initializeState();
```
[the remark this answers is not in the document]
Comment (MohiTheFish) [reply src/scenes/Dashboard/cells/countdown.jsx]: The linter says that if the function is in the class it should use 'this' within the function, and the state just depends on the time.

File: src/scenes/Dashboard/styles/_countdown.scss [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    }
+  } 
+}
+  
+@keyframes animatedflipdowntop {
+  0% {
+    transform: rotateX(0deg);
+  }
+  50%, 100% {
+    transform: rotateX(-91deg);
```
[the remark this answers is not in the document]
Comment (MohiTheFish) [reply src/scenes/Dashboard/styles/_countdown.scss]: With 90 degrees the animation seemed to hang in the air for a moment. With 91, it remains hidden until it's time to 'reveal itself'.

File: src/scenes/Dashboard/cells/sponsors.jsx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    array[currentIndex] = array[randomIndex];
+    array[randomIndex] = temporaryValue;
+  }
+
+  return array;
+}
+
+function Lane(props) {
+  const { myimage } = props;
+  return (
```
[the remark this answers is not in the document]
Comment (MohiTheFish) [reply src/scenes/Dashboard/cells/sponsors.jsx:46]: None taken. I'll change it when those get in.

File: src/scenes/Dashboard/styles/_responsive.scss [hunk 4]
```diff
@@ -0,0 +1,28 @@
+@media (max-width: 500px) {
+  .dashboard-wrapper{
+    .dashboard {
+      .top-row{
+        height: 100vh;
+      }
+      .bottom-row{
```
[the remark this answers is not in the document]
Comment (MuhammedAtNutrien) [reply src/scenes/Dashboard/styles/_responsive.scss]: Resolved.

File: src/scenes/Dashboard/index.jsx [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+import Temp from './cells/tempcell';
+
+// This function will retrieve the new theme that should be applied to the dashboard.
+function getTheme() {
+  const currentTime = new Date();
+  let currentHour = currentTime.getHours();
+  const currentMinute = currentTime.getMinutes();
+  currentHour += currentMinute / 60;
+
+  if (currentHour >= 8 && currentHour < 18) {
```
[the remark this answers is not in the document]
Comment (MohiTheFish) [reply src/scenes/Dashboard/index.jsx]: Looking at this one: https://sunrise-sunset.org/api. 
They say we have to put a link to them if we use their api though.

**Review by brianstrauch — APPROVED** [review]
Merging for the demo tomorrow!!

**Pull request merged**